src/services/category.ts

```diff
@@ -0,0 +1,5 @@
+import {Category} from "../models/Category";
+
+export const getCategories = async (options={}):Promise<Category[]>=>{
```

Comment:
You can add the type of the options: `(options:  FindOptions = {})`. You can get `FindOptions` type from sequelize package.